sale_stock_secondary_unit/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     'name': 'Sale Stock Secondary Unit',
     'summary': 'Get product quantities in a secondary unit',
-    'version': '11.0.1.0.0',
+    'version': '12.0.1.0.0',
     'development_status': 'Beta',
     'category': 'stock',
     'website': 'https://github.com/OCA/stock-logistics-warehouse',
```

Comment:
@digitalsatori URL like `https://github.com/OCA/sale-workflow`

Reply:
Thank you @nikul-serpentcs , changed in the new commit.